src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/WhitelistApiAction.java

```diff
@@ -27,6 +27,7 @@
 import com.amazon.opendistroforelasticsearch.security.securityconf.impl.SecurityDynamicConfiguration;
 import com.amazon.opendistroforelasticsearch.security.ssl.transport.PrincipalExtractor;
 import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
+import com.amazon.opendistroforelasticsearch.security.tools.OpenSearchSecurityAdmin;
```

Comment:
The `@link com.amazon.opendistroforelasticsearch.security.tools.OpenDistroSecurityAdmin` is not working without importing the class.